src/tracing/librbd.tp

```diff
@@ -172,6 +172,32 @@ TRACEPOINT_EVENT(librbd, open_image_exit,
     )
 )
 
+TRACEPOINT_EVENT(librbd, aio_open_image_enter,
```

Comment:
Double-check to see if rbd-replay-prep needs to be updated to handle this new event-type since it explicitly is searching for open/close events.